host/xtest/xtest_20000.c

```diff
@@ -384,6 +386,11 @@ static TEEC_Result obj_corrupt(TEEC_UUID *p_uuid, void *file_id,
 			 ta_dirname, obj_filename);
 
 		get_offs_size(type, block_num, version, &real_offset, NULL);
+		if (real_offset == -1UL) {
```

Comment:
minor: what about adding a return value to `get_offs_size()` (i.e: 0=success, non-0=error). It would be more easy to read/maintain. this is the most common error case detection scheme.